common/src/test/java/tachyon/util/CommonUtilsTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.util;
+
+import java.lang.Thread;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CommonUtilsTest {
+
+  @Test
+  public void getCurrentMsAndSleepMsTest() {
+    long delta = 100;
+    long startTime = CommonUtils.getCurrentMs();
+    CommonUtils.sleepMs(delta);
+    long currentTime = CommonUtils.getCurrentMs();
+
+    /* Check that currentTime falls into the interval [startTime + delta; startTime + 2*delta] */
+    Assert.assertTrue(startTime + delta < currentTime);
+    Assert.assertTrue(startTime + 2 * delta > currentTime);
+  }
+
+  @Test
+  public void testListToString() {
+    class TestCase {
+      List<Object> mInput;
+      String mExpected;
+
+      public TestCase(String expected, Object... objs) {
+        mExpected = expected;
+        mInput = new LinkedList<Object>();
```

Comment:
Minor style comment: Could the array `objs` be converted to a list in one line (using something like `Arrays.asList`)